tools/3D_plasma_initialization_code/readme.txt

```diff
@@ -0,0 +1,22 @@
+author: Pierre Cerantola
+email: cerantolap@eisti.eu
+
+
+Instructions to edit/create a configuration file:
+
+0: you can see "input_config.txt" to have an idea of the right structure to follow
+1: each non-blank line must be a parameter line
+2: each parameter line must follow the structure "numeric_parameter			# comment about numeric_parameter"
```

Comment:
In fact, comment about parameter can be empty, but '#' character is always required, because it is much more simple to read the file by separating terms with "#" character. So the minimum to write in each line is "parameter #".